keyboards/gmmk/pro/config.h

```diff
@@ -53,13 +57,16 @@
 #define SPI_MOSI_PIN A6
 #define SPI_MISO_PIN A7
 
-#define DRIVER_1_CS B13
-#define DRIVER_2_CS B14
-#define DRIVER_1_EN C13
-#define DRIVER_2_EN C13
+#define SPI_SS_DRIVER_1_PIN B13
+#define ENABLE_DRIVER_1_PIN C13
+#define SPI_SS_DRIVER_2_PIN B14
+#define ENABLE_DRIVER_2_PIN C13
+#define RGB_MATRIX_MAXIMUM_BRIGHTNESS 175
+
+#define SW_LINES_ENABLE_DRIVER_1 SW_LINES_ENABLED_1_TO_12
+#define SW_LINES_ENABLE_DRIVER_2 SW_LINES_ENABLED_1_TO_11
```

Comment:
Not a fan of this.

Reply:
Any proposal?

Reply:
Basically, remove it. 

Or more specifically, what is the point of this, other than to remove one of the lines.  In theory, they shouldn't be called by the driver, if they're not configured to, correct? 

If so, what's the point of having this?

Reply:
> 
> 
> Basically, remove it.
> 
> Or more specifically, what is the point of this, other than to remove one of the lines. In theory, they shouldn't be called by the driver, if they're not configured to, correct?
> 
> If so, what's the point of having this?

Done. Please review the last commit.